types/materialize-css/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "ban-types": false,
+    "prefer-method-signature": false
+  }
```

Comment:
@andy-ms Do we allow the new tslint added that disables some of the rules ?

Reply:
Sure, it's better than before at least.